docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+#
+# This file is part of the PyRDP project.
+# Copyright (C) 2018 GoSecure Inc.
+# Licensed under the GPLv3 or later.
+#
+
+version: "3"
+services:
+  pyrdp:
+    build: .
+    environment: 
+      - DISPLAY 
+    network_mode: "host"
```

Comment:
Document why this is needed. It is important to state that if you don't use the graphical interface it is not required. I would also disable it by default. Because if you deploy this tool on the Internet for a honeypot, exposing the whole container to the host network is dangerous.